### PR TITLE
Update samples to SDK version 2.4.0

### PR DESCRIPTION
**Pull request by SatjaSivcev**

### CODE REVIEW

File: source/Applications/Advanced/HandEyeCalibration/README.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ The following applications assume that a **Transformation Matrix** has been foun
 
 [HandEyeCalibration-url]: HandEyeCalibration/HandEyeCalibration.cpp
 [UtilizeEyeInHandCalibration-url]: UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp
-[ZividHandEyeCalibration-url]: https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/96469274
-[Tutorial-url]: https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/72450049
-[PoseConversions-url]: PoseConversions/PoseConversions.cpp
\ No newline at end of file
+[ZividHandEyeCalibration-url]: https://support.zivid.com/latest/rst/academy/applications/hand-eye/hand-eye-calibration-process.html
```
Comment (chrisasc) [inline source/Applications/Advanced/HandEyeCalibration/README.md:38]: should it still be .../rst/...?

**Review by chrisasc — COMMENTED** [review]
Only one comment

File: source/Applications/Advanced/HandEyeCalibration/README.md (hunk 1 above)
Comment (SatjaSivcev) [reply source/Applications/Advanced/HandEyeCalibration/README.md]: done

**Review by chrisasc — APPROVED** [review]
LGTM

**Pull request merged**